.vscode/settings.json

```diff
@@ -9,5 +9,10 @@
   "typescript.preferences.autoImportFileExcludePatterns": [
     "./packages/thirdweb/src/exports"
   ],
-  "typescript.tsdk": "node_modules/typescript/lib"
+  "[typescriptreact]": {
```

Comment:
fix this conflict please, we _do_ want to also keep the `tsdk` field

Reply:
added